bin/main

```diff
@@ -1,3 +1,64 @@
 #!/usr/bin/env ruby
 
-puts 'Hello World!'
+board = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+update_board = []
+
+puts 'Are you ready for a game of Tic Tac Toe?'
+
+puts 'What is your name p1?: '
+player1 = gets.chomp.capitalize
+puts "Welcome #{player1}!!"
+
+puts 'What is your name p2?: '
+player2 = gets.chomp.capitalize
+puts "Nice to meet you #{player2}!"
+
+puts 'This is the game board: '
+puts " #{board[0]} | #{board[1]} | #{board[2]} "
+puts '-----------'
+puts " #{board[3]} | #{board[4]} | #{board[5]} "
+puts '-----------'
+puts " #{board[6]} | #{board[7]} | #{board[8]} "
+
+player_start = [player1, player2].shuffle
+puts "The first one to move will be #{player_start[0]}, you'll be playing with 'X'."
+puts "#{player_start[0]} please make a move, choose from 1 to 9 please."
+player_choice = gets.chomp.to_i
+
+if player_choice > board.length
+  puts 'Invalid position, choose again!'
+elsif board.include?(player_choice)
+  update_board.insert((player_choice - 1), 'X')
+
+  puts " #{update_board[0]} | #{update_board[1]} | #{update_board[2]} "
+  puts '-----------'
+  puts " #{update_board[3]} | #{update_board[4]} | #{update_board[5]} "
+  puts '-----------'
+  puts " #{update_board[6]} | #{update_board[7]} | #{update_board[8]} "
+
+end
```

Comment:
As required on the first review, the user's move should be inside a loop.
You can create something like this, and then adjust it in Milestone 3:
```game_on = true

# initial instructions for player

while game_on
  # loop for each move
  
  if winner # and/or draw (the exact condition in this milestone is not important)
    game_on = false
  end
end
```